pytripgui/field_vc/field_cont.py

```diff
@@ -47,10 +54,33 @@ def set_model_from_view(self):
         else:
             self.model.isocenter = []
 
-        self.model.gantry = self.view.gantry_angle
-        self.model.couch = self.view.couch_angle
+        self.model.display_angles_in_standard = self.view.angles_standard
+        self.model.gantry, self.model.couch = self._get_view_angles_in_trip_standard()
+
         self.model.fwhm = self.view.spot_size
         self.model.raster_step = self.view.raster_step
         self.model.dose_extension = self.view.dose_extension
         self.model.contour_extension = self.view.contour_extension
         self.model.zsteps = self.view.depth_steps
+
+    def _get_view_angles_in_trip_standard(self):
+        _gantry, _couch = self.view.gantry_angle, self.view.couch_angle
```

Comment:
Please introduce consistent naming across model and view. I'd propose to use:
`self.model.gantry_angle_trip98`, `self.view.gantry_angle` (as it could store angle in both formats)

Right now we have `model.gantry` and `view.gantry_angle`. It is not clear why `_angle` suffix is added only to the last one.